torch/distributions/transforms.py

```diff
@@ -16,6 +16,7 @@
     'LowerCholeskyTransform',
     'PowerTransform',
     'SigmoidTransform',
+    'TanhTransform',
```

Comment:
Can't this be achieved using:
- AffineTransform(0, 2) (gives (2x))
- SigmoidTransform (gives (sigma(2x)))
- AffineTransform(2, 1) (gives (2sigma(2x) - 1) = tanh(x))

Reply:
@vishwakftw Indeed. However, it seems the sampled action is different under same random seed. 

```python
dist = TransformedDistribution(Normal(torch.zeros(3), torch.ones(3)), [TanhTransform()])

torch.manual_seed(0)

action = dist.sample()
print(action)

>>> tensor([ 0.9123, -0.2853, -0.9747])
```

and 

```pythion
transforms = [AffineTransform(0, 2), SigmoidTransform(), AffineTransform(2, 1)]
dist2 = TransformedDistribution(Normal(torch.zeros(3), torch.ones(3)), transforms)
dist2

torch.manual_seed(0)

action = dist2.sample()
print(action)

>>> tensor([2.9561, 2.3574, 2.0126])
```

Reply:
Sorry - it's supposed to read `AffineTransform(-1, 2)` (because -1 + 2x).

Reply:
Thanks a lot @vishwakftw ! This is very neat solution, also since currently PyTorch doesn't support `arctanh` yet, the way you proposed can work around the numerical instability of naive `arctanh` computation.